packages/api-logs/jest.config.js

```diff
@@ -0,0 +1,9 @@
+const config = require('../../jest.config');
+
+module.exports = {
+  ...config,
+  rootDir: './',
+  coveragePathIgnorePatterns: [
+    'lib/'
```

Comment:
Can you remove this now that we dont have the Oas dependency?

Reply:
Removed